test/release/examples/primers/syncsingle.chpl

```diff
@@ -21,76 +21,19 @@ config const n = 7;
 var sy$: sync int=1;  // state = full, value = 1
 var si$: single bool; // state = empty, value = false
 
-// By convention, sync and single variable names end in a ``$`` to
-// add a visual cue indicating that reads and writes are possibly
-// expensive operations and can potentially block the task.
+// As a convention, sync and single variable names sometimes end in a
+// ``$`` to add a visual cue indicating that reads and writes are possibly
+// expensive operations that can potentially block the task.
```

Comment:
Not related to this PR, but I wonder whether we'll relax this convention in the near (or distant) future given that accidentally referring to a sync is less problematic (and all methods on variables can be potentially expensive).